src/main/java/com/cegeka/academy/service/mapper/ChallengeMapper.java

```diff
@@ -93,5 +93,17 @@ public static ChallengeCategoryDTO convertChallengeCategoryToChallengeCategoryDT
 
     }
 
+    public static Challenge enrichChallenge(ChallengeDTO challengeDTO, Challenge challenge)
+    {
+        challenge.setStartDate(challengeDTO.getStartDate());
+        challenge.setEndDate(challengeDTO.getEndDate());
+        challenge.setPoints(challengeDTO.getPoints());
+        challenge.setDescription(challengeDTO.getDescription());
+        challenge.setStatus(challengeDTO.getStatus());
+        challenge.setChallengeCategory(convertChallengeCategoryDTOToChallengeCategory(challengeDTO.getChallengeCategory()));
+
```

Comment:
In cazul ChallengeCategory, nu este aceeasi situatie? Daca obiectul challenge are o categorie tot enrich trebuie sa faci, in cazul in care nu are faci convert

Reply:
Sigur. Imediat ce ajung acasa